src/openforms/authentication/contrib/yivi_oidc/admin.py

```diff
@@ -1,17 +1,46 @@
 from django.contrib import admin
 
+from import_export import resources
+from import_export.admin import ImportExportModelAdmin
+
 from .models import AttributeGroup
 
 
+class AttributeGroupResource(resources.ModelResource):
+    class Meta:
+        model = AttributeGroup
+        # Use uuid as identifier
+        import_id_fields = ("uuid",)
+        fields = ("name", "uuid", "description", "attributes")
+
+    def dehydrate_name(self, obj: AttributeGroup) -> str:
+        # Trimming whitespace
+        return obj.name.strip()
+
+    def dehydrate_description(self, obj: AttributeGroup) -> str:
+        # Trimming whitespace
+        return obj.description.strip()
+
+
 @admin.register(AttributeGroup)
-class AttributeGroupAdmin(admin.ModelAdmin):
+class AttributeGroupAdmin(ImportExportModelAdmin):
+    import_template_name = "admin/forms/attributegroup/import_form.html"
+    export_template_name = "admin/forms/attributegroup/export_form.html"
```

Comment:
why are these template overrides necessary? If it's for the django-admin-index stuff, aren't there some base templates to override instead?